collectors/cgroups.plugin/cgroup-network.c

```diff
@@ -230,7 +230,6 @@ static struct ns {
 };
 
 int switch_namespace(const char *prefix, pid_t pid) {
-    if(!prefix) prefix = "";
 
```

Comment:
The fix is wrong. The line should be moved inside the `ifdef`

Reply:
It's not wrong cause the same check is being done again inside the `ifdef` by `proc_pid_fd()`. It's redundant code.

Reply:
Right. Perhaps I wasn't clear enough in my commit message?

Reply:
Nope, just too hasty from my part.